tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# Azure Functions localsettings file
```

Comment:
Do we really need an isolated gitignore file?

Reply:
Good call. The [repository's .gitignore file](https://github.com/Azure/azure-sdk-tools/blob/master/.gitignore) already contains the same entries along with a few extras for VS that artifacts aren't in here.

Reply:
They are just part of the template that VS lays down. There are a few critical files that we must not check in (e.g. local.settings.json). I tend to accept the default to avoid shooting myself in the foot. I'll double check the root has everything and submit a PR taking them out.